### ISSUE TITLE
Azure TSI Create issue - 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by MatchingRadar** [Azure/azure-cli-extensions#2003]
## Create TSI issue - https://docs.microsoft.com/en-us/cli/azure/ext/timeseriesinsights/timeseriesinsights/environment/standard?view=azure-cli-latest#ext-timeseriesinsights-az-timeseriesinsights-environment-standard-create

**Command Name**
`az timeseriesinsights environment standard create
Extension Name: timeseriesinsights. Version: 0.1.2.`

**Errors:**
```
'NoneType' object is not iterable
Traceback (most recent call last):
pip-install-363drsgw\knack\knack\cli.py, ln 215, in invoke
cli\core\commands\__init__.py, ln 654, in execute
cli\core\commands\__init__.py, ln 718, in _run_jobs_serially
cli\core\commands\__init__.py, ln 711, in _run_job
Temp\pip-install-363drsgw\six\six.py, ln 703, in reraise
...
    partition_key_properties=[TimeSeriesIdProperty(name=id_property, type="String") for id_property in partition_key_properties]
TypeError: 'NoneType' object is not iterable
```

## To Reproduce:
- `az timeseriesinsights environment standard create -g {} --name {} --location {} --sku-name {} --sku-capacity {} --data-retention-time {} --storage-limit-exceeded-behavior {}`

## Expected Behavior
TSI environment is created

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.6
Installer: MSI

azure-cli 2.8.0

Extensions:
azure-iot 0.9.1
databricks 0.2.0
stream-analytics 0.1.0
timeseriesinsights 0.1.2

```

**Comment by yonzhan** [comment]
add to S173

**Comment by jiasli** [comment]
Thanks for the finding. Fixed in #2353.
